Sources/CoreMetrics/SystemMetrics.swift

```diff
@@ -0,0 +1,172 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Metrics API open source project
+//
+// Copyright (c) 2018-2019 Apple Inc. and the Swift Metrics API project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift Metrics API project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Combine
+import Foundation
+
+@available(OSX 10.15, *)
```

Comment:
why is this needed?

Reply:
The scheduler that returns a `Cancellable` requires Combine, which in turn needs 10.15, but like @Mordil left in a comment that's not available on Linux. So we probably have to re-think that part entirely. Might be possible to use a `Timer` instead, though not sure how those are with thread safety.

Reply:
how about using DispatchWorkItem or DispatchOperation?